Columbus/frontend/src/components/Dialogs/EditProfileDialog/EditProfileDialog.js

```diff
@@ -74,18 +74,16 @@ export default function EditProfileDialog(props) {
     const data = new FormData(document.getElementById("edit-form"));
 
     let birthday = null
-    try 
+    if(dateValue && isDateChanged) 
     {
+      console.log(dateValue)
```

Comment:
there is a console log

Reply:
It was due to the resolving the conflicts in wrong manner. It should be fixed